src/main/java/uk/gov/justice/hmpps/prison/repository/jpa/repository/PrisonerActivitiesCountRepository.kt

```diff
@@ -28,10 +29,9 @@ interface PrisonerActivitiesCountRepository : CrudRepository<PrisonerActivitiesC
           AND CS.SCHEDULE_DATE >= :startDate
           AND CS.SCHEDULE_DATE <= :endDate
         WHERE CA.AGY_LOC_ID = :agencyId 
-          AND OB.AGY_LOC_ID = :agencyId
           AND OPP.AGY_LOC_ID = :agencyId
           AND (OPP.OFFENDER_PROGRAM_STATUS = 'ALLOC'
-             OR (OPP.OFFENDER_PROGRAM_STATUS = 'END' AND OPP.OFFENDER_END_DATE >= :startDate))
+             OR (OPP.OFFENDER_PROGRAM_STATUS = 'END' AND OPP.OFFENDER_END_DATE >= :endDate))
```

Comment:
this is an existing bug in the old stats code.  Will fix in a subsequent PR, but for now will reproduce existing behaviour so that can compare the old and new stats queries